contracts/Cloudwalk/TransientBorrowRateModel.sol

```diff
@@ -150,21 +149,6 @@ contract TransientBorrowRateModel is InterestRateModel {
         owner = newOwner;
     }
 
-    /**
-     * @dev Transfers ownership of the contract to a new account (`newOwner`)
-     * Can only be called by the current owner
-     */
-    function updateBlocksPerYear(uint256 newBlocksPerYear)
```

Comment:
After removing this function the following constant should be removed too:
* `blocksPerYearMax`
* `blocksPerYearMin`

Reply:
We will remove `TransientBorrowRateModel` contract in the future. Changes reverted.